app/controllers/WorkerController.php

```diff
@@ -2,23 +2,41 @@
 
 namespace GanttDashboard\App\Controllers;
 
+use GanttDashboard\App\Validators\Traits;
+use GanttDashboard\App\Validators\Worker as WorkerValidator;
+use GanttDashboard\App\Models\Workers as WorkerModel;
 use GanttDashboard\App\Services\Worker as WorkerService;
 use Phalcon\Http\Response;
 use Phalcon\Http\ResponseInterface;
 
 class WorkerController extends ControllerBase
 {
+    use Traits;
```

Comment:
avoid traits. create a service/class that validates, and send your post object there